Doc/library/email.utils.rst

```diff
@@ -65,6 +65,11 @@ of the new API.
    *email address* parts.  Returns a tuple of that information, unless the parse
    fails, in which case a 2-tuple of ``('', '')`` is returned.
 
+   If *strict* is true, use a strict parser which rejects malformed inputs.
+
+   .. versionchanged:: 3.13
```

Comment:
If it is backported, it should be 3.12.1.

Reply:
Right, the version number should be updated in each backport. I removed the "needs backport" label to avoid automated backports.

Reply:
I agree with the concept but that depends on the backport situation. It is fair to start with 3.13 here and if we do decide to backport it further as a security fix later on, the docs can be updated to say 3.12.x.